README.md

```diff
@@ -1,15 +1,188 @@
-Welcome to your new dbt project!
+# Avo Audit
 
-### Using the starter project
+Avo audit helps you detect issues in your raw analytics event tables by querying a sample of the raw events and comparing them against each other.
+This allows you to find discrepancies between events and properties that should look the same, but don’t.
 
-Try running the following commands:
-- dbt run
-- dbt test
+## Issue types
 
+Issues available in the alpha version:  
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
-- Check out [Discourse](https://discourse.getdbt.com/) for commonly asked questions and answers
-- Join the [chat](http://slack.getdbt.com/) on Slack for live discussions and support
-- Find [dbt events](https://events.getdbt.com) near you
-- Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+**Property sometimes missing** – detects absence of event properties across tracking calls.  
+>`Property onboarding status is not sent with 30% of the App Opened event.`  
+
+**Property type mismatch** – detects inconsistent event property types across tracking calls.  
+>`Inconsistent type of property onboarding status on event App Opened: int (84%), string (16%).`
```

Comment:
```suggestion
>_Property onboarding status is not sent with 30% of the App Opened event._ 

**Property type mismatch** – detects inconsistent event property types across tracking calls.  
>_Inconsistent type of property onboarding status on event App Opened: int (84%), string (16%)._
```

Reply:
Yeah I agree this looks a bit too heavy on formatting now